src/pages/LinkDevicePage/LinkDevicePage.tsx

```diff
@@ -24,15 +24,15 @@ const LinkDevicePage: FC = () => {
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
   const [siteId, setSiteId] = useState<Maybe<Site["id"]>>(null);
   const [searchQuery, setSearchQuery] = useState<string>("");
-  const [selectedDevices, setSelectedDevices] = useState<Device[]>([]);
+  const [selectedDevices, setSelectedDevices] = useState<DeviceType[]>([]);
   const { isLoading, isFetching, sites, devices } = useSearchDevices(siteId, searchQuery);
-  const dpUserId = useMemo(() => get(context, ["data", "user", "id"]), [context]);
+  const dpUserId = useMemo(() => context?.data?.user.id, [context]);
```

Comment:
>Unnecessary optional chain on a non-nullish value
`context` is always an `object`

Reply:
I'm afraid context doesn't always is object.
<img width="318" alt="Screenshot 2024-07-24 at 16 28 53" src="https://github.com/user-attachments/assets/17de8bbc-7c99-41d7-b0d4-e9a508279f9a">

Reply:
Here story for fix types https://app.shortcut.com/deskpro/story/158090/app-sdk-should-fix-context-typings